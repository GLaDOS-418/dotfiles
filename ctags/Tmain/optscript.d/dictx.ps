0 dict == clear
0 dict dup /x 1 put == clear
1 dict ==
1 dict dup /y 2 put == clear
2 dict ==
2 dict dup /z 2 put == clear

{
    { -1 dict } stopped {
        _errorname =
        stop
    } if
} stopped pop
_errorname =

<< >> == clear
<<>> dup /x 0 put == clear

<< /x 0 >> ==
<</x 0>> dup /y 1 put == clear

<< /x 0 /y 1 >> ==
<< /x 0 /y 1 >> dup /z 2 put == clear

{
    { >> } stopped {
               _errorname =
               stop
           } if
}  stopped pop

{
    { << /a >> } stopped {
        _errorname =
        stop
    } if
} stopped pop
_errorname =
