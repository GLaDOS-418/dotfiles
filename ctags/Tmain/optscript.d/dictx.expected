-dict:0-
-dict:1-
-dict:0-
-dict:1-
-dict:0-
-dict:1-
rangecheck
null
-dict:0-
-dict:1-
-dict:1-
-dict:2-
-dict:2-
-dict:3-
unmatchedmark
rangecheck
null
